javascript/javascript1/week2/EventApplication.js

```diff
@@ -0,0 +1,9 @@
+const todayDate = new Date()
+const day = todayDate.getDay()
+const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+function getEventWeekday(eventInDays) {
+    const eventDay = (eventInDays + day)%7
+    return eventDay
```

Comment:
This is an index of the day. You can return the day instead `return weekday[eventDay]`

Reply:
okay.. i fixed it.. thanks